vignettes/rhino-fluent.Rmd

```diff
@@ -0,0 +1,321 @@
+
+Upon running the app with `shiny::runApp()`, you should see the following on your screen!
+
+```{r, echo=FALSE}
+knitr::include_graphics("images/rhino_fluent_1.png")
+```
+
+## Adding modules to the Rhino App
+
+Your application runs, but it doesn't have any meaningful functionality.
+Let's add something there!
+
+In Rhino, each application view is intended to live as a [Shiny module](https://shiny.rstudio.com/articles/modules.html) and use encapsulation provided by [`box` package](https://klmr.me/box/).
+
+We will now try to code a module that basically creates a checkbox and shows values once checked or unchecked.
+
+Rhino already created a good place for new modules, the `app/view` directory.
+So, create a file there, named `checkbox.R` and paste the following code there -
+
+``` r
+# app/view/checkbox.R
+
+box::use(
+  shiny[moduleServer, NS, tags, textOutput, renderText, markdown], # Load shiny package and specific functions
+  shiny.fluent[fluentPage, Checkbox.shinyInput]  # Load shiny.fluent package and specific functions
+)
+
+#' @export
+ui <- function(id) {
+  ns <- NS(id)  # Create namespace for the app
+  fluentPage(  # Create a Fluent UI page
+    markdown("The `checkbox` module works! 🎉"),  # Add markdown text to the page
+    tags$h2(  # Create a heading
+      Checkbox.shinyInput(ns("checkbox"), value = TRUE),  # Add a checkbox input with a default value of TRUE
+      textOutput(ns("checkboxValue"))  # Add a text output that will display the checkbox value
+    )
+  )
+}
+
+#' @export
+server <- function(id) {
+  moduleServer(id, function(input, output, session) {  # Define a module server function that takes in input, output, and session arguments
+    output$checkboxValue <- renderText({  # Render the text output based on the input checkbox value
+        sprintf("Value: %s", input$checkbox)
+      })  
+  })
+}
+```
+
+Within the **`ui`** function, we use **`fluentPage`** to create a Fluent UI page.
+We then add a **`markdown`** text element to the page, followed by an **`h2`** heading that contains a checkbox input and a text output element.
+
+The checkbox input is created using the **`Checkbox.shinyInput`** function from the **`shiny.fluent`** package.
+We give the checkbox a unique ID by passing **`ns("checkbox")`** as the first argument, which uses the namespace we created earlier.
+We also set a default value of **`TRUE`** for the checkbox.
+
+The text output element is created using the **`textOutput`** function from the **`shiny`** package.
+We give it a unique ID using **`ns("checkboxValue")`**, which again uses the namespace we created earlier.
+
+The `server` all these UI components once the `checkbox` module is called by the main script.
+So, now we will need to call the `checkbox` module in the `app/main.R` file.
+Notice how we have modified the code that we wrote earlier to get started -
```

Comment:
Again, you explain literally everything, while what is needed is to explain how to use shiny.fluent.